develop/dev-guide-third-party-tools-compatibility.md

```diff
@@ -0,0 +1,158 @@
+
+### TiDB does not maintain `Com_*` counters
+
+**Description**
+
+MySQL maintains a series of [server status variables starting with `Com_`](https://dev.mysql.com/doc/refman/8.0/en/server-status-variables.html#statvar_Com_xxx) to keep track of the total number of operations you have performed on the database. For example, `Com_select` records the total number of `SELECT` statements initiated by MySQL since it was last started (even if the statements were not queried successfully). TiDB does not maintain these variables. You can use the statement `SHOW GLOBAL STATUS LIKE 'Com_%'` to see the difference between TiDB and MySQL.
+
+**Way to avoid**
+
+Please do not use these variables. One of the common usage scenarios is monitoring. TiDB is well observable and does not require querying from server status variables. For custom monitoring tools, refer to [TiDB Monitoring Framework Overview](/tidb-monitoring-framework.md).
+
+### TiDB distinguishes between `TIMESTAMP` and `DATETIME` in error messages
+
+**Description**
+
+TiDB error messages distinguish between `TIMESTAMP` and `DATETIME`, while MySQL does not, and returns them all as `DATETIME`. That is, MySQL incorrectly converts `TIMESTAMP` type error messages to `DATETIME` type.
+
+**Way to avoid**
+
+Do not use the error messages for string matching, but use [Error Codes](/error-codes.md) for troubleshooting.
+
+### TiDB does not support the `CHECK TABLE` statement
+
+**Description**
+
+The `CHECK TABLE` statement is not supported in TiDB. To check the consistency of data and corresponding indexes, you can use the [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) statement in TiDB.
+
+**Way to avoid**
+
+Use [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) instead.
+
+## MySQL JDBC incompatibility
+
+Test Version: MySQL Connector/J `8.0.29`
+
+### Default collations are inconsistent
+
+**Description**
+
+The collations of MySQL Connector/J are stored in the client and are distinguished by the server version.
+
+Known character sets for which client-side and server-side collations do not match:
+
+| Character Sets | Client Default Collations | Server Default Collations |
+| - | - | - |
+| `ascii` | `ascii_general_ci` | `ascii_bin` |
+| `latin1` | `latin1_swedish_ci` | `latin1_bin` |
+| `utf8mb4` | `utf8mb4_0900_ai_ci` | `utf8mb4_bin` |
+
+**Way to avoid**
+
+Set the sorting rules manually and do not rely on the client default sorting rules (client default sorting rules are saved by the MySQL Connector/J configuration file).
+
+### The `NO_BACKSLASH_ESCAPES` parameter is ineffective
+
+**Description**
+
+The `NO_BACKSLASH_ESCAPES` parameter cannot be used so that `\` character is not escaped. Tracking the [issue](https://github.com/pingcap/tidb/issues/35302).
```

Comment:
```suggestion
The `NO_BACKSLASH_ESCAPES` parameter cannot be used so that `\` character is not escaped. Track this [issue](https://github.com/pingcap/tidb/issues/35302).
```